package.json

```diff
@@ -46,6 +46,7 @@
   "dependencies": {
     "history": "^3.0.0",
     "lodash.assign": "^4.2.0",
+    "lodash.isequal": "^4.4.0",
```

Comment:
Can we use `fbjs`'s `shallowEqual` instead? Since the query object's values are always strings, we don't have to check for deep equality. Since React includes `fbjs`, this also means that most consumers won't have to add anything to their bundle.